x-pack/plugins/security/server/authorization/privileges/feature_privilege_builder/cases.ts

```diff
@@ -17,12 +17,14 @@ const readOperations: string[] = [
   'getComment',
   'getTags',
   'getReporters',
+  'getUserActions',
```

Comment:
Does caseStatus need to be added as well or no

Reply:
Naa, case status is just a fancy find all cases under the hood aka a `getCase`. So we'll use `getCase` for it.